package.xml

```diff
@@ -2,29 +2,31 @@
   <name>srdfdom</name>
   <version>0.3.1</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
-  <author email="isucan@willowgarage.edu">Ioan Sucan</author>
+  <author email="isucan@willowgarage.com">Ioan Sucan</author>
   <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
-  <maintainer email="isucan@willowgarage.com">Ioan Sucan</maintainer>
+  <maintainer email="isucan@gmail.com">Ioan Sucan</maintainer>
```

Comment:
I just emailed @isucan to see if he wants to remain maintainer for packages like this, I'm fine either way